src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.lang.reflect.Array;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Application {
+    FigureSupplier figureSupplier = new FigureSupplier();
+    public static void main(String[] qwe) {
+        Figure [] figure = {new Square(),new Rectangle(),new RightTriangle(),
+                new IsoscelesTrapezoid()};
```

Comment:
Creating instances of figures directly in the array is not recommended. The checklist suggests generating random values in 'FigureSupplier' and using them to create figure instances.